pkgs/applications/science/biology/star/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "star";
```

Comment:
Include version into the name attribute. Nix parses package version from name attribute and `version` one has no special meaning.